pkg/network/nmstate.go

```diff
@@ -0,0 +1,40 @@
+package network
+
+import (
+	"os"
+	"path/filepath"
+	"reflect"
+
+	"github.com/kubevirt/cluster-network-addons-operator/pkg/render"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+func changeSafeNMState(prev, next *opv1alpha1.NetworkAddonsConfigSpec) []error {
+	if prev.NMState != nil && !reflect.DeepEqual(prev.NMState, next.NMState) {
+		return []error{errors.Errorf("cannot modify NMState Daemon configuration once it is deployed")}
+	}
+	return nil
+}
+
+// renderNMState generates the manifests of NMState handler
+func renderNMState(conf *opv1alpha1.NetworkAddonsConfigSpec, manifestDir string, clusterInfo *ClusterInfo) ([]*unstructured.Unstructured, error) {
+	if conf.LinuxBridge == nil {
+		return nil, nil
+	}
+
+	// render the manifests on disk
+	data := render.MakeRenderData()
+	data.Data["NMStateHandlerImage"] = os.Getenv("NMSTATE_HANDLER_IMAGE")
```

Comment:
maybe just `NMSTATE_IMAGE` without the `HANDLER` ?

Reply:
actually this data and environment variable should be nmstate state handler image

Reply:
so this becomes wordy, at least as far as I understand the comment,  we end up with something like NMStateState in multiple places.  could we just abbreviate it for the future of having nmstate and nmpolicy?  I'll update the PR per my understanding of the comment.

Reply:
It is a little too wordy, but I prefer that over implicit names. It's hidden from user anyway.